pkg/apiserver/registry/stats/nodelatencystats/rest.go

```diff
@@ -50,38 +58,96 @@ func (r *REST) Destroy() {
 }
 
 func (r *REST) Create(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
-	// TODO: fill this function in next PR
-	return &statsv1alpha1.NodeLatencyStats{}, nil
+	summary := obj.(*statsv1alpha1.NodeLatencyStats)
+	if err := r.indexer.Update(summary); err != nil {
```

Comment:
add the following comment above this line:

```go
// Update will add the object if the key does not exist.
```

Reply:
Ok.